Vagrantfile

```diff
@@ -99,9 +108,20 @@ Vagrant.configure(2) do |config|
         shell.privileged = false
       end
 
-      if machine.key?(:deploylatest) and machine[:deploylatest]
+      # Should machine provisioning include deploying montagu, either latest or named branch?
+      if machine.key?(:deploymontagu) and ['latest','branch'].include? machine[:deploymontagu]
         machine_config.vm.provision :shell do |shell|
-          shell.path = 'staging/shared/deploy-latest-montagu'
+          if machine[:deploymontagu] == 'latest'
+            shell.args = 'latest'
+          end
+
+          if machine[:deploymontagu] == 'branch'
+            # Expect this vagrant environment variable to be set to desired branch on host machine
+            # (or allow default to latest branch)
+            shell.args = [ENV["BRANCH"] || '']
```

Comment:
Using BRANCH rather than VERSION as it seemed more descriptive of what it does

Reply:
strictly I think this could be "ref" because one could use a sha, branch or tag!  How much do you think that matters?

Reply:
Ha yeah, fair point. VERSION was better after all!

Reply:
How about using `REF`?